db/seeds.rb

```diff
@@ -8,15 +8,58 @@
 #   movies = Movie.create([{ name: 'Star Wars' },
 # { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+#
+#
+#
+require 'faker'
+
+Answer.destroy_all
```

Comment:
You shouldn't delete anything in seeds, that's wrong. `seeds.rb` appends the data, it doesn't replace it